src/app/modules/project/overview/project/project.component.html

```diff
@@ -0,0 +1,92 @@
+   but WITHOUT ANY WARRANTY; without even the implied warranty
+   of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+   See the GNU Lesser General Public License for more details.
+
+   You can find a copy of the GNU Lesser General Public License
+   along with this program, in the LICENSE.md file in the root project directory.
+   If not, see https://www.gnu.org/licenses/lgpl-3.0.txt
+
+ -->
+
+<!-- List project -->
+<ng-container *ngIf="showListView">
+  <div class="project list">
+    <div class='circle' [ngStyle]="{'background-image': 'url(' + getIconUrl(project) + ')'}">
+    </div>
+    <div class="project-info">
+      <h3 class="project-name">{{project.name}}</h3>
+      <p class="project-short-description">{{project.shortDescription}}</p>
+    </div>
+    <div class="item project-likes"><em class='far fa-heart'></em> 0 Likes</div>
+    <div class="item disabled project-created-date" [tooltip]="projectUpdated">
+      <em class='far fa-clock'></em>
+      {{project.created | date: 'longDate'}}
+      <ng-template #projectUpdated>
+        Created: {{project.created | date: 'mediumDate'}}<br>
+        Updated: {{project.updated | date: 'mediumDate'}}
+      </ng-template>
+    </div>
+    <div class="item project-user-name"><em class='fas fa-user'></em> {{project.user.name}}</div>
+  </div>
+</ng-container>
+
+<!-- Grid Project-->
+<ng-container *ngIf="!showListView">
+  <div class="project grid">
+    <div class="project-info">
+      <div class='top-wrapper'>
+        <h3 class="project-name">{{project.name}}</h3>
+        <div class='circle' [ngStyle]="{'background-image': 'url(' + getIconUrl(project) + ')'}">
+        </div>
+      </div>
+      <p class="project-short-description">{{project.shortDescription}}</p>
+    </div>
+    <div class='tags'>
+      <div class='tag' (click)="tagClicked($event)">UX</div>
+      <div class='tag' (click)="tagClicked($event)">UI</div>
+      <div class='tag' (click)="tagClicked($event)">Development</div>
+      <div class='tag' (click)="tagClicked($event)">Creatief</div>
+      <div class='tag' (click)="tagClicked($event)">Arduino</div>
+      <div class='tag' (click)="tagClicked($event)">HTML</div>
+      <div class='tag plus' (click)="tagClicked($event)">+ 3</div>
+    </div>
+    <div class='bottom-wrapper'>
+      <div class='item'>
+        <em class='far fa-heart'></em>
+        <div class="project-likes">0 Likes</div>
+      </div>
+      <div [tooltip]="project.collaborators && project.collaborators.length > 0 ? projectCollaborators : null" class='item'>
+        <ng-template #projectCollaborators>
+            <ul>
+              <li *ngFor="let collaborator of project.collaborators">
+                {{collaborator.fullName}}
+              </li>
+            </ul>
+        </ng-template>
+        <em class='fas fa-users'></em>
+        <div class="project-collaborators">{{project.collaborators ? project.collaborators.length : 0}} collaborators</div>
```

Comment:
The collaborator view could use some minor improvements for instance:
- Center the names in the tooltip
- Remove the bullet points and perhaps add a small horizontal line in between collaborators? What does @liekevdvoort think about this?

Reply:
![image](https://user-images.githubusercontent.com/49523639/100606782-3f529b80-330a-11eb-85f8-91446cea4cd9.png)
I would propose to do it like in the design of the project detail